Troubleshooting.md

```diff
@@ -0,0 +1,21 @@
+﻿# Build Troubleshooting
+Below are some tips for troubleshooting common issues.
+
+### Docker-Related Errors
+If you receive an error similar to:
+```
+C:\Program Files\Docker\Docker\Resources\bin\docker.exe: Error response from daemon: driver failed programming external connectivity on endpoint redisTestContainer (bbafe9e2b9e88b14e1d231eae755ae33755292e1399a278a846cab5fdc6ae955): Error starting userland proxy: mkdir /port/tcp:0.0.0.0:6379:tcp:172.17.0.2:6379: input/output error.
+```
+
+Or other Docker-related errors, it may be necessary to update your version of Docker. Try updating to the latest version of docker and attempting to build again.
+
+### Loading in Visual Studio 
+The command-line build makes changes to the file system that will sometimes prevent you from successfully loading all of the projects in the solution when you open the solution in Visual Studio.
```

Comment:
Rather than the script, I think I'd prefer to simply indicate that you need to have `%USERPROFILE%\.dotnet\x64` on the `PATH` when launching VS. People are welcome to build scripts for their own use.

Reply:
This whole section can be removed in lieu of a simple statement about the PATH. There's no need to go deep into explaining why this is necessary